### PR TITLE
Add Note about the borked 2.x version in the README

### PR DESCRIPTION
**Pull request by animaux**
;)

### CODE REVIEW

**Review by nitriques — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 A simple way to manipulate images "just in time" via the URL. Supports caching, image quality settings and loading of offsite images.
 
+-----
+
+**NOTE: Version 2.x is not working properly, please use the 1.x branch until further notice!**
+
```
Comment (nitriques) [inline README.markdown]: Can you add this link https://github.com/symphonycms/jit_image_manipulation/tree/1.x please ?

**Review by nitriques — CHANGES_REQUESTED** [review]

**Review by nitriques — APPROVED** [review]

**Comment by brendo** [comment]
Hmm, how can I help here? [master...integration](https://github.com/symphonycms/jit_image_manipulation/compare/integration) has 104 commits difference, surely `integration` can be released?

**Comment by andrewminton** [comment]
HAs this been tested with php7.3 at all?

**Comment by animaux** [comment]
@andrewminton Version 2 is still a deadend. JIT 1.46 runs fine on PHP 7.3.

**Comment by andrewminton** [comment]
Thanks for info!

**Comment by animaux** [comment]
My pleasure ;)

**Pull request merged**